diff --git a/config.def.h b/config.def.h
index 0e01717..c288227 100644
--- a/config.def.h
+++ b/config.def.h
@@ -82,33 +82,35 @@ char *termname = "st-256color";
  */
 unsigned int tabspaces = 8;
 
-/* Terminal colors (16 first used in escape sequence) */
+/*
+ * Terminal colors (16 first used in escape sequence)
+ * Base16 st template by Daniel Mulford
+ * Dark Violet scheme by ruler501 (https://github.com/ruler501/base16-darkviolet)
+ */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+    /* 8 normal colors */
+	[0] = "#000000", /* black   */
+	[1] = "#a82ee6", /* red     */
+	[2] = "#4595e6", /* green   */
+	[3] = "#f29df2", /* yellow  */
+	[4] = "#4136d9", /* blue    */
+	[5] = "#7e5ce6", /* magenta */
+	[6] = "#40dfff", /* cyan    */
+	[7] = "#b08ae6", /* white   */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	[8]  = "#593380", /* black   */
+	[9]  = "#a82ee6", /* red     */
+	[10] = "#4595e6", /* green   */
+	[11] = "#f29df2", /* yellow  */
+	[12] = "#4136d9", /* blue    */
+	[13] = "#7e5ce6", /* magenta */
+	[14] = "#40dfff", /* cyan    */
+	[15] = "#a366ff", /* white   */
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
@@ -118,8 +120,8 @@ static const char *colorname[] = {
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 7;
+static unsigned int defaultrcs = 0;
 
 /*
  * Default shape of cursor
